litho-it/src/test/java/com/facebook/litho/widget/RecyclerBinderWrapContentTest.java

```diff
@@ -60,16 +58,15 @@
 
   private ComponentContext mComponentContext;
   private ShadowLooper mLayoutThreadShadowLooper;
-  private RecyclerView mRecyclerView;
+
+  private TestRecyclerView mRecyclerView;
 
   @Before
   public void setup() throws Exception {
     mComponentContext = new ComponentContext(RuntimeEnvironment.application);
-    mRecyclerView = mock(RecyclerView.class);
+    mRecyclerView = new TestRecyclerView(mComponentContext.getAndroidContext());
 
-    mLayoutThreadShadowLooper =
-        Shadows.shadowOf(
-            (Looper) Whitebox.invokeMethod(ComponentTree.class, "getDefaultLayoutThreadLooper"));
+    mLayoutThreadShadowLooper = Shadows.shadowOf((Looper) Whitebox.invokeMethod(ComponentTree.class, "getDefaultLayoutThreadLooper"));
```

Comment:
Can you reformat it as it was, please? Line is too long.

Reply:
Done